docs/guides/docs/running-a-node/index.mdx

```diff
@@ -31,6 +31,16 @@ Running your own node provides several advantages:
 1. **Query Freedom:** By hosting your own node, you can execute a higher number of queries without encountering any rate limits.
 2. **Network Independence:** Having your own node ensures that you're not reliant on third-party services, giving you full control over your interactions with the Fuel blockchain.
 
+## Minimum Requirements
+
+| Hardware    | Minimum Requirement | Minimum Requirement (Low Traffic) |
```

Comment:
```suggestion
| Hardware    | Minimum Requirement | Minimum Requirement (Low Traffic) |
```

Do we have an estimate numerical value for what is considered Low Traffic? Could be helpful to have some estimated number representation here.